app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -71,11 +47,19 @@ To be eligible for a bursary, you’ll need at least one of the following:
 
 If your course combines 2 subjects (for example, English and history), at least 50% of your course content must be made up of the bursary subject to be eligible.
 
-You’ll also need to meet the bursary scheme’s individual terms and conditions.
+#### How they're paid
```

Comment:
```suggestion
#### How bursaries are paid
```
Currently there are two identical h4s on the page which is not good for screen reader users